bin/radical-pilot-create-static-ve

```diff
@@ -112,11 +112,16 @@ mkdir -p "$prefix"
 cd $prefix
 
 echo -n "install virtualenv "
-VIRTENV_TGZ="virtualenv-1.9.tar.gz"
-VIRTENV_TGZ_URL="https://pypi.python.org/packages/source/v/virtualenv/$VIRTENV_TGZ"
-curl -k -L -O "$VIRTENV_TGZ_URL" 2>&1 | progress
-tar zxmf "$VIRTENV_TGZ"
-VIRTENV_CMD="$PYTHON virtualenv-1.9/virtualenv.py"
+
+VIRTENV_CMD="$(which virtualenv 2>/dev/null)"
```

Comment:
Isn't this change coming from devel?

Reply:
I re-synced with `feature/summit`, this is clean now.